### PR TITLE
Refactor lamassu-admin React project entrypoint

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Create a customized react admin](https://trello.com/c/SPZYxR2U/611-create-a-customized-react-admin)

File: new-lamassu-admin/src/pazuz/apollo/Provider.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  process.env.NODE_ENV === 'development' ? 'https://localhost:8070' : ''
+
+const getClient = (history, location) =>
+  new ApolloClient({
+    link: ApolloLink.from([
+      onError(({ graphQLErrors, networkError }) => {
+        if (graphQLErrors)
+          graphQLErrors.forEach(({ message, locations, path, extensions }) => {
+            if (extensions?.code === 'UNAUTHENTICATED') {
+              if (location.pathname !== '/404') history.push('/404')
```
Comment (hellypi) [inline new-lamassu-admin/src/pazuz/apollo/Provider.js:20]: can't we have this inside a single if?

**Review by hellypi — CHANGES_REQUESTED** [review]

File: new-lamassu-admin/src/pazuz/apollo/Provider.js (hunk 1 above)
Comment (chaotixkilla) [reply new-lamassu-admin/src/pazuz/apollo/Provider.js:20]: Wouldn't that lead to repeated code? As there's code left to execute inside the first `if` statement.

Comment (hellypi) [reply new-lamassu-admin/src/pazuz/apollo/Provider.js:20]: you're right, missed it!

**Review by hellypi — APPROVED** [review]

**Review by RafaelTaranto — APPROVED** [review]

File: new-lamassu-admin/src/index.js [hunk 2]
```diff
@@ -1,10 +1,28 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 
-import App from './App'
 import * as serviceWorker from './serviceWorker'
 
-ReactDOM.render(<App />, document.getElementById('root'))
+function importBuildTarget() {
+  if (process.env.REACT_APP_BUILD_TARGET === 'LAMASSU') {
+    return import('./App')
```
Comment (joshmh) [inline new-lamassu-admin/src/index.js]:
```suggestion
    return import('./lamassu/App')
```

For consistency.

**Review by joshmh — CHANGES_REQUESTED** [review]

**Pull request merged**